oracle.py

```diff
@@ -32,23 +32,32 @@ def __init__(self, config, logger):
     def init_oracle(self):
         # calling the relevant oracle. All oracles should follow the model of OracleBase
         if self.config.oracle.main == "mlp":
-            self.oracle = OracleMLP(self.config)
+            self.proxy = OracleMLP(self.config)
```

Comment:
why calling an oracle a proxy ??

Reply:
so that the acquisition function remains consistent for all kinds of acquisitions. I could elaborate this better over a call if it isn't clear right now